dev/com.ibm.ws.jpa.tests.spec10.embeddable_fat/test-applications/embeddable/nested/src/com/ibm/ws/jpa/embeddable/nested/model/Embeddable01.java

```diff
@@ -0,0 +1,78 @@
+package com.ibm.ws.jpa.embeddable.nested.model;
```

Comment:
Missing the standard header, this is the case for all of these model classes by the looks of it.